security/ransomware/README.md

```diff
@@ -6,40 +6,19 @@ With this Workshop, you'll learn:
 
 # Disclaimer
 
-This Workshop is for educational purposes only. Poc Innovation isn't responsable
+:warning: This Workshop is for educational purposes only. Poc Innovation isn't responsable
 for what you will do with this knowledge.
 
-We provided you a `Document/` directory to infect. Be careful not to infect your own and important files.
+We provided you a `Documents/` directory to infect. Be careful not to infect your own and important files.
 If you are afraid to do so, please do this workshop on a virtual machine.
 
-# Installation
```

Comment:
Add a line to add the setup like [this](https://github.com/PoCInnovation/Workshops/tree/master/software/14.Vercel#step-0---setup)